### ISSUE TITLE
Unclear what or where rest-schema is

### ISSUE THREAD

**Issue by Relequestual** [riverbed/reschema#1]
Found this repo via https://github.com/json-schema-org/json-schema-spec/issues/29
Curious to know what "rest-schema" is. Can you provide any additional info or links?

Thanks

**Comment by handrews** [comment]
rest-schema == reschema
https://support.riverbed.com/apis/steelscript/reschema/toc.html
see also
https://support.riverbed.com/apis/steelscript/sleepwalker/toc.html

I should submit a pull request to change the reference to GL4v2 to point to these docs.
reschema and sleepwalker are both installable with pip.

**Comment by Relequestual** [comment]
OK Thanks.

It looks somewhat proprietary, but seeminly well documented.
Does it have librariers in other languges?

**Comment by Relequestual** [comment]
**judging a project by the number of stars it has on github**
=/

**Comment by handrews** [comment]
I'm not aware of any other language versions of the system.  @mgarabed any plans for JavaScript or anything else?  I've started poking at JSON Schema again which is how this discussion started.

> judging a project by the number of stars it has on github

 @Relequestual : I starred it just now, so it's obviously twice as good as was five minutes ago :-)
In fairness, I don't think these libraries have been promoted outside of the Riverbed SteelScript system even thought they could stand alone.
